gradle.properties

```diff
@@ -37,6 +37,13 @@ botania_version=1.18.2-432
 cc_tweaked_version=1.100.5
 immersive_engineering_version=1.18.2-8.0.2-pre.12
 ctm_version=1.1.2.6
+cofh_core_curse_id = 4112120
```

Comment:
Can you also update `mods.toml` to require the min version of CoFH which uses IMC for registering holding compat?

Reply:
I can, but that will make the mod unusable with CoFH Core until CoFH Core releases version 9.1.2 (which is the version the IMC compat is added in). They removed the old way of registering for Holding enchantments a few versions ago, and only have the IMC way as part of one of their latest versions which is only on their Github right now.

Reply:
Ah, I thought that had already been released. OK, as long as it doesn't actually crash with the old version (i.e. just does nothing), that should be alright.